src/protocols/rdp/keyboard.c

```diff
@@ -248,37 +446,21 @@ static guac_rdp_key* guac_rdp_keyboard_get_key(guac_rdp_keyboard* keyboard,
  *     The keymap to use to populate the given client's keysym/scancode
  *     mapping.
  */
-static void __guac_rdp_keyboard_load_keymap(guac_rdp_keyboard* keyboard,
+static void guac_rdp_keyboard_load_keymap(guac_rdp_keyboard* keyboard,
```

Comment:
If this is still `static`, and thus presumably only used within the context of this file, why the change in name?

Reply:
Apparently, the double underscore prefix is actually a reserved namespace in C. It's supposed to be dedicated to the use of the compiler. Having learned this, I've been switching use of `__guac_*` to `guac_*` wherever I encounter it.